extensions/guacamole-auth-duo/src/main/java/org/apache/guacamole/auth/duo/api/SignedDuoCookie.java

```diff
@@ -221,7 +221,7 @@ private static String sign(String key, String data) throws GuacamoleException {
             mac.init(new SecretKeySpec(key.getBytes("UTF-8"), SIGNATURE_ALGORITHM));
 
             // Return signature as hex
-            return DatatypeConverter.printHexBinary(mac.doFinal(data.getBytes("UTF-8"))).toLowerCase();
+            return BaseEncoding.base16().encode(mac.doFinal(data.getBytes("UTF-8"))).toLowerCase();
```

Comment:
Guava provides a mechanism for controlling the case of the resulting string prior to encoding:

https://google.github.io/guava/releases/27.0-jre/api/docs/com/google/common/io/BaseEncoding.html#lowerCase--

Fairly minor, but probably worth switching over to `BaseEncoding.base16().lowerCase()`.

Reply:
Okay, swapped this over to lower prior to encoding.